system/data/es/systems/core.json

```diff
@@ -0,0 +1,270 @@
+{
+    "name": "Core (Básico)",
+    "identifier": "core",
+    "types": {
+        "Aspects": [
+            {
+                "label": "Básico",
+                "entries": [
+                    {
+                        "name": "Concepto principal",
+                        "type": "aspect",
+                        "data": {
+                            "label": "Concepto principal"
+                        }
+                    },
+                    {
+                        "name": "Complicación",
+                        "type": "aspect",
+                        "data": {
+                            "label": "Complicación"
+                        }
+                    },
+                    {
+                        "name": "Aspecto vacío",
```

Comment:
Those "Aspecto vacío"s are named "Otro aspecto" in Condensed & Accelerated, and also below in the Opcional option, shouldn't they be the same here?

Reply:
It's like this in the English version, maybe it has to do with that in Fate core character creation has 5 aspects by default (High concept, trouble, and 3 from the Phase trio).